README.md

```diff
@@ -1 +1,44 @@
-# RESTAURANT-PAGE
\ No newline at end of file
+# Restaurant Page
+
+## Description
+
+This is a JavaScript project from Odin's curriculum. It is based on DOM manipulation by dynamically rendering a simple restaurant homepage.
+There I used JavaScript alone to generate the entire contents of the website.
+
+## Built With
+
+- JavaScript
+- Webpack
+
+## Set up this projecect locally
+
+- Open the terminal in the directory where you want to install the app;
+- Run command `git clone git@github.com:FrederickMih/RESTAURANT-PAGE.git` which will clone the project;
+- Switch to the main directory of project using cd;
+- Run `npm install webpack webpack-cli --save-dev` in your computer terminal to install Webpack (you will need Node.js installed on your machine);
+- Open index.html file in your browser;
+
+
+## Screenshot
+
+| ![](assets/images/shot1.png) |
+ 
+
+### Deployment
+
+This application is deployed on [GitHubPages](https://frederickmih.github.io/RESTAURANT-PAGE/).
```

Comment:
- [ ] This link is displaying a page which does not have stylings and images as you coded the project. Kindly fix it.

|Your project on local machine|Deployed version|
|---|---|
|![screencapture-localhost-5500-dist-2021-05-30-09_35_16](https://user-images.githubusercontent.com/56886064/120096235-863f1380-c12a-11eb-9ccc-9912c5c5f95a.png)|![screencapture-frederickmih-github-io-RESTAURANT-PAGE-2021-05-30-09_37_40](https://user-images.githubusercontent.com/56886064/120096265-c4d4ce00-c12a-11eb-99c6-968c7d6ecd2a.png)|